packages/core/src/client.ts

```diff
@@ -1885,7 +1885,18 @@ export class MedplumClient extends EventTarget {
       xhr.setRequestHeader('Cache-Control', 'no-cache, no-store, max-age=0');
       xhr.setRequestHeader('Content-Type', contentType);
       xhr.setRequestHeader('X-Medplum', 'extended');
-      xhr.send(data);
+      if (data instanceof ReadableStream) {
+        const writableStream = new WritableStream({
+          write(chunk) {
+            xhr.send(chunk);
+          },
+        });
+
+        // eslint-disable-next-line no-void
+        void data.pipeTo(writableStream);
```

Comment:
```suggestion
        data.pipeTo(writableStream).catch(console.error);
```
I think doing this should make the `void` here unnecessary and will make sure we don't have any floating promises

Reply:
gotcha ya wasn't exactly sure what to do here given that resolve/reject is being handled in `xhr.onload`
can drop this in